From 68f9deead3ac327dadd096e004d24a6166f57158 Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan@agner.ch>
Date: Wed, 27 Apr 2022 20:36:19 +0200
Subject: [PATCH] ARM: dts: bcm2711: yellow: use generic activity trigger for
 green LED

Use the generic trigger "activity" for the green LED so that any
system activity is shown.

Signed-off-by: Stefan Agner <stefan@agner.ch>
---
 arch/arm/boot/dts/broadcom/bcm2711-rpi-cm4-ha-yellow.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/broadcom/bcm2711-rpi-cm4-ha-yellow.dts b/arch/arm/boot/dts/broadcom/bcm2711-rpi-cm4-ha-yellow.dts
index 3866192a6f477..11da9ec9b828a 100644
--- a/arch/arm/boot/dts/broadcom/bcm2711-rpi-cm4-ha-yellow.dts
+++ b/arch/arm/boot/dts/broadcom/bcm2711-rpi-cm4-ha-yellow.dts
@@ -602,7 +602,7 @@ &leds {
 	act_led: led-act {
 		label = "led0";
 		default-state = "off";
-		linux,default-trigger = "mmc0";
+		linux,default-trigger = "activity";
 		gpios = <&gpio 42 GPIO_ACTIVE_HIGH>;
 	};
 
